install/gcp/scripts/initialize_forseti_services.sh

```diff
@@ -49,7 +49,7 @@ CONFIG_VALIDATOR_COMMAND+=" -port=50052"
 sudo chmod ugo+x ${FORSETI_HOME}/external-dependencies/config-validator/ConfigValidatorRPCServer
 
 SQL_PROXY_COMMAND="$(which cloud_sql_proxy)"
-SQL_PROXY_COMMAND+=" -instances=${SQL_INSTANCE_CONN_STRING}=tcp:${SQL_PORT}"
+SQL_PROXY_COMMAND+=" -instances=${SQL_INSTANCE_CONN_STRING}=tcp:${SQL_PORT} -ip_address_types=PRIVATE"
```

Comment:
Based on the documentation, the correct way of specifying this option is to use double dashes "--ip_address_types=PRIVATE", can you add another dash to the option?

https://cloud.google.com/sql/docs/mysql/sql-proxy

Reply:
I have tested with `-` and `--` dashes. They both work. From `cloud_sql_proxy --help` i got this:
```
  -ip_address_types
    Default to be 'PUBLIC,PRIVATE'. Options: a list of strings separated by ',', e.g. 'PUBLIC,PRIVATE'
```

I have reported this discrepancy as in issue on Documentation link you provided above.

TODO:
1. I will change to `--` because this is still official documentation.
2. Deployment Templates need to be updated to set Cloud SQL with private IP. I think that at the moment CloudSQL is set only with Public IP.

Reply:
Following up on your comment, would you mind also adding the changes to the deployment template to use private IP?

Reply:
i don't think that my fix is correct. `-ip_address_types` is necessary if cloudSQL is setup with PRIVATE+PUBLIC ip. If cloudSQL is setup with only PRIVATE or only PUBLIC ip than there is no need for `-ip_address_types`